src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -100,22 +100,33 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         // In AzDO, the path to the system chromedriver is in an env var called CHROMEWEBDRIVER
         // We want to use this because it should match the installed browser version
         // If the env var is not set, then we fall back on using whatever is in the Selenium config file
-        var chromeDriverArg = string.Empty;
         var chromeDriverPathEnvVar = Environment.GetEnvironmentVariable("CHROMEWEBDRIVER");
         if (!string.IsNullOrEmpty(chromeDriverPathEnvVar))
         {
-            chromeDriverArg = $"--javaArgs=-Dwebdriver.chrome.driver={chromeDriverPathEnvVar}/chromedriver";
             output.WriteLine($"Using chromedriver at path {chromeDriverPathEnvVar}");
+            var chromeDriverVersion = new ProcessStartInfo
+            {
+                FileName = chromeDriverPathEnvVar,
+                Arguments = "--version",
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+            };
+            Process.Start(chromeDriverVersion).WaitForExit();
```

Comment:
This should make sure we print the version of the Chromedriver on the CI machine

Reply:
/cc @SteveSandersonMS